openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -128,6 +141,14 @@ def assert_block_is_gated(self, block, is_gated):
                 # check that has_access did not raise the IncorrectPartitionGroupError thereby not gating the block
                 self.assertFalse(mock_access_error.called)
 
+    def test_lti_audit_access(self):
```

Comment:
why make this a separate test from the other problem types?

Reply:
LTI problem types have an extra field I need to set and I thought this was an easier way to do it. Do you think it'd be better to pass in the extra field to the problem types test in some way? I thought this was more clear.

Reply:
i guess it is a matter of personal preference so this is fine